ppa/archive/admin.py

```diff
@@ -59,6 +60,30 @@ def source_link(self, obj):
     source_link.admin_order_field = 'source_id'
     source_link.allow_tags = True
 
+    def save_model(self, request, obj, form, change):
+        '''Note any fields in the protected list that have been changed
+        and preserve in database.'''
+        # new object, created from scratch, nothing to track and preserve
+        # NOTE: Is this logic correct? Or should all the fields be preserved?
+        if not change:
+            super().save_model(request, obj, form, change)
+            return
+        db_obj = DigitizedWork.objects.get(pk=obj.pk)
```

Comment:
`DigitizedWork` extends `TrackChangesModel` - is there any reason we can't use the `has_changed` method here?

Reply:
I think I have a way to use this using `refresh_from_db`; the wrinkle as discussed being the obj is a new creation and not aware of the database. If that doesn't work, I'll note the problem.

Reply:
No dice on `has_changed` and `refresh_from_db`. In the end, cleanest way is to compare against a DB copy. I've added a note explaining why I'm not using `has_changed`.